integration_tests/e2e/signIn.cy.ts

```diff
@@ -21,36 +21,36 @@ context('Sign In', () => {
 
   it('User name visible in header', () => {
     cy.signIn()
-    const indexPage = Page.verifyOnPage(IndexPage)
-    indexPage.headerUserName().should('contain.text', 'J. Smith')
+    const contactsPage = Page.verifyOnPage(ContactsPage)
+    contactsPage.headerUserName().should('contain.text', 'J. Smith')
   })
 
   it('User name visible in header', () => {
```

Comment:
can this test be removed as it is same as the previous test ?

Reply:
Yes, I duplicated by mistake. It's been removed now